Test/ParallelPrimitives/RadixSort.h

```diff
@@ -10,7 +11,14 @@ namespace Oro
 class RadixSort
 {
   public:
-	typedef unsigned int u32;
+	using u32 = uint32_t;
+	using u64 = uint64_t;
+
+	struct KeyValueSoA
```

Comment:
SoA (Structure of Array) is used here instead of AoS (Array of Structure) to make the execution more efficient on GPU.